dutree/dutree.py

```diff
@@ -467,35 +481,31 @@ def human(value):
     return '{}   B'.format(value)
 
 
-def main():
-    pathname = None
-    if len(sys.argv) == 2:
-        def getsize(node):
-            return node.app_size()
-
-        pathname = sys.argv[1]
-        use_apparent_size = True
-
-    elif len(sys.argv) == 3:
-        def getsize(node):
-            return node.use_size()
+def main() -> None:
+    parser = argparse.ArgumentParser(prog='dutree', description='dutree shows a summary of the directories/files which take up the most space.')
+    parser.add_argument('--count-blocks', help='count block size not app size', action='store_true')
```

Comment:
I'd like to replace this with using this as a default, and then use `--apparent-size` for the current default.

BUT, that is for a _separate changeset_.

So, this is fine for now.

Reply:
can do that later if you want